contracts/protocol/facets/PauseHandlerFacet.sol

```diff
@@ -86,7 +86,7 @@ contract PauseHandlerFacet is ProtocolBase, IBosonPauseHandler {
 
             for (uint256 i = 0; i < totalRegions; i++) {
                 // Check if the region is paused by bitwise AND operation with shifted 1
-                if ((status.pauseScenario & (1 << i)) != 0) {
+                if (status.pauseScenario & (1 << i) != 0) {
```

Comment:
I personally prefer explicit statement with braces. 
However, in the logic of this PR, and according to https://docs.soliditylang.org/en/v0.8.17/cheatsheet.html#order-of-precedence-of-operators, the bitwise shift "<<" will be resolved before the bitwise AND "&".
So these remaining braces are still useless
```suggestion
                if (status.pauseScenario & 1 << i != 0) {
```

Reply:
You are right.
I applied the change, but the prettier put these parentheses back.

I cannot find the setting to turn it off. And I think in this place, although they are logically not needed, they actually add to the legibility of the code. 

Also, when it comes to the combination of bitwise shift and some other bitwise operator, we wrap it in parentheses in other places in the code.

So if you agree, I would just keep it as it is now.

Reply:
no pb